### ISSUE TITLE
aten::cudnn_convolution_backward_weight not found 

### ISSUE THREAD

**Issue by iszihan** [nv-tlabs/GET3D#96]
Hello, I was trying to run the training file but got this error after following some fixes with this #91 

aten::cudnn_convolution_backward_weight not found when calling from ops/conv2d_grad_fix.py line 179. After searching around, it seems to suggest we need to replace this function with aten::convolution_backward (according to https://github.com/pytorch/pytorch/issues/74437), which requires a shuffling of input arguments and some additional arguments such as output mask, and bias size. I'm wondering if you know another fix and if not, how should one modify the arguments to convolution_backward() for this call? Thank you!

**Comment by SteveJunGao** [comment]
Hi @iszihan,

I guess this is related to the version of pytorch, are you using pytorch 1.9.0?

**Comment by Bathsheba** [comment]
You're on the right track, keep following

**Comment by SteveJunGao** [comment]
Hi @iszihan @Bathsheba,

any followups about this issue?

**Comment by iszihan** [comment]
I'm on pytorch 1.13.0, and currently the code does run with the following modification though I'm not 100% sure if this does what the old code do.

`# General case => cuDNN.
            name = 'aten::convolution_backward'
            #flags = [torch.backends.cudnn.benchmark, torch.backends.cudnn.deterministic, torch.backends.cudnn.allow_tf32]
            op, _ = torch._C._jit_get_operation(name)
            empty_weight = torch.tensor(0.0, dtype=input.dtype, device=input.device).expand(weight_shape)
            output_mask = [False,True,False]
            _, grad_weight, _ = op(grad_output, input, empty_weight, None, stride, padding, dilation, transpose, (0,0), groups, output_mask) #*flags)
            return grad_weight`
